forms/src/app/login-model-form/login-model-form.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="container ph4 pb4 black-80 br3 ma3 center w-70">
+  <p class="f3">Login (Template based)</p>
+  <form #formRef="ngForm" (ngSubmit)="onSubmit(formRef.value); formRef.reset()" class="measure center">
+    <div class="mt3">
+      <label for="userid" class="db fw6 lh-copy f5">Username</label>
+      <input type="text"
+             name="userid"
+             id="userid"
+             [(ngModel)]="username"
```

Comment:
Two data binding an input element is just fine for this application, though it may be a good idea to include an example of single way binding, using the `[formContro]` directive for inputs. That way if Augury in the future is able to incorporate two-data-binding and non-two-way-data-binding into some sort of feature, we have our bases covered.